File: programme/fibonnaci-series-using-recursion/fibonaci-recursion.py
# Write a programme to print the Fibonacci series for 'n' numbers

def Fib(n):
    if n == 0:
        return 0
    if n == 1:
        return 1
    else:
        return (Fib(n-1)+Fib(n-2))


n = int(input("Input: "))
print('Output: ', end='')
for i in range(n):
    print(Fib(i), end=' ')

File: programme/fibonnaci-series-using-recursion/fibonaci-recursion.c
// Write a programme to print fibonacci series using recursion
#include <stdio.h>

int fibonacci(int);

int main()
{
    int n, i;
    printf("Input  : ");
    scanf("%d", &n);
    printf("Output : ");
    for (i = 0; i < n; i++)
    {
        printf("%d ", fibonacci(i));
    }
    printf("\n");
    return 0;
}

int fibonacci(int i)
{
    if (i == 0)
        return 0;
    else if (i == 1)
        return 1;
    else
        return (fibonacci(i - 1) + fibonacci(i - 2));
}
